res2df/nnc.py

```diff
@@ -1,5 +1,6 @@
 """
-Extract non-neighbour connection (NNC) information from Eclipse output files.
+Extract non-neighbour connection (NNC) information from reservoir
+simulator output files.
```

Comment:
Could use `term` here. Also, NNC will most probably always mean that it is a `EGRID` file.